packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -1718,13 +1718,15 @@ Viewer.prototype.isDestroyed = function () {
  */
 Viewer.prototype.destroy = function () {
   let i;
-
-  this.screenSpaceEventHandler.removeInputAction(
+  if(defined(this.screenSpaceEventHandler)){
```

Comment:
A more thorough fix would also be to check if the screenspace event handler is defined, but destroyed:

```suggestion
  if(defined(this.screenSpaceEventHandler) && !this.screenSpaceEventHandler.isDestroyed()){
```

Reply:
Thanks for the suggestion @ggetz! I've updated and run `npm run prettier` as well.